Sources/ForageSDK/Component/ForagePANTextField/ForagePANTextField.swift

```diff
@@ -218,54 +252,72 @@ public class ForagePANTextField: UIView, Identifiable {
 // MARK: - UITextFieldDelegate
 
 extension ForagePANTextField: UITextFieldDelegate {
+    public func textFieldDidBeginEditing(_ textField: UITextField) {
+        delegate?.focusDidChange(self)
+    }
+    
+    public func textFieldDidEndEditing(_ textField: UITextField) {
```

Comment:
My understanding is that the `textFieldDidEndEditing` handler is equivalent to [a blur event](https://developer.apple.com/documentation/uikit/uitextfielddelegate/1619591-textfielddidendediting). @dleis612 was this intentional? I want to make sure that this can't cause GoPuff to misread a blur event as a focus event and cause a wonky experience for users.